packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -638,6 +642,7 @@ export class ContainerRuntime
 		containerScope: FluidObject = context.scope,
 		existing?: boolean,
 		containerRuntimeCtor: typeof ContainerRuntime = ContainerRuntime,
+		initializeEntryPoint?: (containerRuntime: IContainerRuntime) => Promise<FluidObject>,
```

Comment:
don't add this to the deprecated method

Reply:
Good point; if consumers are going to make changes anyway to start leveraging it, might as well have them move to the new method.